spec/scripts/string_spec.rb

```diff
@@ -8,6 +8,18 @@
           "Expected 'string_multiplication.rb' to NOT print the String literal 'HoHoHo', but did."
       end
     end
+
+    multiplication_counter = 0
+    File.foreach(multiplication_file).with_index do |line, line_num|
+      if line.include?("*") 
+        unless line.include?("#")
+          multiplication_counter += 1
+        end
+      end
+    end
+    expect(multiplication_counter).to be >= 1,
+      "Expected 'string_addition.rb' to use the * multiplication method, but only #{multiplication_counter} '*' found."
```

Comment:
I wonder if this error message can be improved at all 🤔 This message will only really show up if they haven't used `*` at all, which means `multiplication_counter` is always `0`. I feel like the way the message is phrased, it sounds like `multiplication_counter` could be another number and still be wrong, when that's not really the case.

> Expected 'string_addition.rb' to use the * multiplication method, but only 0 '*' found

Maybe the word "only" feels a little off to me here. What do you think about removing that word?

Reply:
I agree. Let's drop the only. Thanks for the feedback.